storage/innobase/ut/ut0crc32.cc

```diff
@@ -96,6 +96,9 @@ ut_crc32_func_t	ut_crc32_legacy_big_endian;
 but very slow). */
 ut_crc32_func_t	ut_crc32_byte_by_byte;
 
+/** Text description of CRC32 implementation */
+const char *ut_crc32_implementation = NULL;
```

Comment:
I would omit the `= NULL` here.
According to the C standard, global variables are zero-initialized and put into the BSS segment by default. On some systems, adding the redundant zero-initializer can cause the data to be stored and `memcpy()`ed at startup.

A minor nitpick is that in InnoDB code, the `*` should be preceded by no space and succeeded by TAB.